apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -48,6 +48,15 @@ private async Task<int> InsertAsync(SpannerParameterCollection values)
             }
         }
 
+        private SpannerConnection GetConnection() => _fixture.GetConnection();
+
+        private async Task<SpannerDataReader> GetWriteTestReader(SpannerConnection connection)
+        {
+            using var command = connection.CreateSelectCommand($"SELECT * FROM {_fixture.TableName} WHERE K=@k");
```

Comment:
Same here, this using is not correct here.

These tests might all be working just becuase we pull so little data that we only query the service once. But still...